### ISSUE TITLE
Handle citations

### ISSUE THREAD

**Issue by hinkelman** [hinkelman/chez-docs-scrape#1]
Citations should be removed

```
<a name="./binding:s22"></a>Fluid bindings are similar to
<i>special</i> bindings in Common Lisp&nbsp;[<a class=citation href="[./bibliography.html#g181](view-source:https://cisco.github.io/ChezScheme/csug9.5/bibliography.html#g181)">30</a>], except that
(1) there is a single namespace for both lexical and fluid bindings, and
(2) the scope of a fluidly bound variable is not necessarily global.
```

**Comment by hinkelman** [comment]
See also `(doc "run-cp0")`

**Comment by hinkelman** [comment]
Addressed in commit [6fb0395](https://github.com/hinkelman/chez-docs-scrape/commit/6fb0395fe6c6af714d600d8fe6b6e53231cdd077).
